### PR TITLE
Fix yast2_nfs_server modue not works after service check in migration regression test

### PR DESCRIPTION
**Pull request by hjluo**
We have yastr2-nfs-server check in the service check modules in migration's regression test, 
so we need to do cleanup in the service check after migration and check if nfsv4 was enabled
before start the NFS server in yast2_nfs_server module.

- Related ticket: https://progress.opensuse.org/issues/90815
- Needles: N/A
- Verification run: 
  https://openqa.nue.suse.com/tests/5924341 
  https://openqa.nue.suse.com/tests/5924666
  https://openqa.nue.suse.com/tests/5961025

### CODE REVIEW

File: lib/nfs_common.pm [hunk 1]
```diff
@@ -219,8 +219,13 @@ sub config_service {
...
     # Disable NFSv4
-    send_key 'alt-v';
-    wait_still_screen 1;
+    do {
+        assert_screen([qw(nfsv4-disabled nfsv4-enabled)], 120);
+        if (match_has_tag('nfsv4-enabled')) {
+            send_key 'alt-v';
+            wait_still_screen 1;
+        }
+    } while (not match_has_tag('nfsv4-disabled'));
```
Comment (lemon-suse) [inline lib/nfs_common.pm:228]: Why need the loop? I think multi-tag is enough.

File: lib/nfs_common.pm [hunk 2]
```diff
@@ -271,8 +276,11 @@ sub check_y2_nfs_func {
         start_service($rw, $ro);
     }
     check_service($rw, $ro);
-    if ($stage eq 'before') {
-        stop_service();
+    stop_service();
+    if ($stage eq 'after') {
```
Comment (lemon-suse) [inline lib/nfs_common.pm:280]: Please add comments to describe why need these cmds.

File: lib/nfs_common.pm (hunk 1 above)
Comment (hjluo) [reply lib/nfs_common.pm:228]: will try if it works. thanks

File: lib/nfs_common.pm (hunk 2 above)
Comment (hjluo) [reply lib/nfs_common.pm:279]: done

File: lib/nfs_common.pm (hunk 1 above)
Comment (hjluo) [reply lib/nfs_common.pm]: works and changed the code https://openqa.nue.suse.com/tests/5961025#step/install_service/43

**Review by lemon-suse — APPROVED** [review]

**Comment by DimStar77** [comment]
> > Needles: N/A
> 
> I doubt this is very true:
> 
> https://openqa.opensuse.org/tests/1729285#step/yast2_nfs_server/69
> 
> This test now fails on TW

Created the missing needle (could do the one for 'enabled') and restarted the test

**Pull request merged**